kbase-extension/static/kbase/js/widgets/appWidgets2/common.js

```diff
@@ -28,8 +30,8 @@ define([], () => {
         );
     }
 
-    function containerContent(div, button, events, ui, container, input) {
-        const clipboard = clipboardButton(div, button, events, ui, container);
+    function containerContent(div, button, events, ui, container, input, copyFn) {
```

Comment:
Future work could just have this little module make its own calls to `common/html` to make `div` and `button` functions happen. Leaving as-is for now, though it's maybe a little odd.